lib/Runtime/Language/ModuleNamespace.cpp

```diff
@@ -111,4 +143,220 @@ namespace Js
         // keep a local copy of the module/
         unambiguousNonLocalExports->AddNew(propertyId, *nonLocalExportNameRecord);
     }
+
+    BOOL ModuleNamespace::HasProperty(PropertyId propertyId)
+    {
+        SimpleDictionaryPropertyDescriptor<BigPropertyIndex> propertyDescriptor;
+        const Js::PropertyRecord* propertyRecord = GetScriptContext()->GetThreadContext()->GetPropertyName(propertyId);
+        if (propertyRecord->IsSymbol())
+        {
+            return this->DynamicObject::HasProperty(propertyId);
+        }
```

Comment:
that's right :)

---
In reply to: [74168450](https://github.com/Microsoft/ChakraCore/pull/1401#discussion_r74168450) [](ancestors = 74168450,74163949)